### PR TITLE
Add powerMonitor events

### PR DESCRIPTION
**Pull request by maddie**

### CODE REVIEW

**Comment by maddie** [comment]
Tested OK locally with changes in asticode/go-astilectron#364

File: index.js [hunk 1]
```diff
@@ -41,6 +41,44 @@ function onReady () {
         client.write(consts.targetIds.app, consts.eventNames.displayEventRemoved, {displays: {all: screen.getAllDisplays(), primary: screen.getPrimaryDisplay()}})
     })
 
+    const powerMonitor = electron.powerMonitor
+    // Listen to power events
+    powerMonitor.on('suspend', function() {
+        client.write(consts.targetIds.power, consts.eventNames.powerEventSuspend)
```
Comment (asticode) [inline index.js]: As mentioned in the go PR, could you remove the power target id and use the app's one instead?

**Review by asticode — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (maddie) [reply index.js]: Done

**Comment by asticode** [comment]
I've made a `v0.54.0` tag

Cheers

**Pull request merged**